src/components/table/table-header/table-header.js

```diff
@@ -119,6 +119,37 @@ class TableHeader extends React.Component {
   }
 
   /**
+   * Returns descriptive text to describe the sortable column, and about
+   * what will happen when they next attempt to sort the column i.e. which
+   * direction it will sort in.
+   *
+   * NB If the current sortOrder is undefined, assume the next sort order
+   * will be descending.
+   *
+   * @method sortDescription
+   * @return {string}
+   */
+  get sortDescription() {
+    if (!this.props.sortable) {
+      return null;
+    }
+
+    let currentSortOrder = this.context.sortOrder;
```

Comment:
Left this as is - `one-var` is set to `off` in our `.eslintrc` so it should be a problem.